src/rpc/methods/state.rs

```diff
@@ -1441,6 +1442,60 @@ impl RpcMethod<1> for StateCompute {
     }
 }
 
+pub enum StateCompute {}
+
+impl RpcMethod<3> for StateCompute {
+    const NAME: &'static str = "Filecoin.StateCompute";
+    const PARAM_NAMES: [&'static str; 3] = ["height", "messages", "tipsetKey"];
+    const API_PATHS: ApiPaths = ApiPaths::V1;
+    const PERMISSION: Permission = Permission::Read;
+    const DESCRIPTION: Option<&'static str> =
+        Some("A flexible command that applies the given messages on the given tipset");
+
+    type Params = (ChainEpoch, Vec<Message>, ApiTipsetKey);
+    type Ok = ComputeStateOutput;
+
+    async fn handle(
```

Comment:
It might make sense to add this command to the filter list, given it can be quite costly. https://docs.forest.chainsafe.io/guides/methods_filtering#public-rpc-node-recommendations

Reply:
Done.